/oak:
  ros__parameters:
    camera:
      i_nn_type: rgb
      i_pipeline_type: rgb
    nn:
      i_nn_config_path: depthai_ros_driver/segmentation
      i_enable_passthrough: true

